examples/goldilocks_models/find_goldilocks_models.cc

```diff
@@ -61,10 +62,14 @@ DEFINE_int32(N_sample_tr, -1, "Sampling # for turning rate");
 DEFINE_bool(is_zero_touchdown_impact, false,
             "No impact force at fist touchdown");
 DEFINE_bool(is_add_tau_in_cost, true, "Add RoM input in the cost function");
+DEFINE_bool(is_uniform_grid, true, "Uniform grid of task space");
+DEFINE_bool(is_restricted_sample_number, false, "Restrict the number of samples. This makes sense"
+                                               "only when is_uniform_grid=false");
 
 // inner loop
-DEFINE_string(init_file, "", "Initial Guess for Trajectory Optimization. "
-                             "E.g. w0.csv");
```

Comment:
Why deleting this line?

Reply:
I didn't delete this line and I checked that it was still in the code. I am not sure why it indicates that I deleted it.

Reply:
The code in github doesn't have this line. Are you sure you are working on the same branch?

Reply:
https://github.com/DAIRLab/dairlib/blob/84a7a3598d6807f890f067fdf75d6df6c91e7ac0/examples/goldilocks_models/find_goldilocks_models.cc#L70

Reply:
Right, we can see from your link that it doesn't have `"E.g. w0.csv"`

Reply:
Oh, it might be deleted by mistake while dealing with the conflicts during merging with your code.